src/tools/pontoon/notifications.md

```diff
@@ -0,0 +1,61 @@
+# Notifications
+
+Pontoon can send internal notifications to users for specific events, or when manually triggered by Administrators.
+
+When the user receives a notification, the bell icon near the user avatar in the top right corner will show a badge with the number of unread notifications.
+
+![Bell icon with 1 unread notification](../../assets/images/pontoon/notifications/unread_notifications.png "Bell icon with a badge showing 1 unread notification")
+
+Clicking the bell icon will display the last notifications in a dropdown menu.
+
+![Dropdown notifications](../../assets/images/pontoon/notifications/notifications_dropdown.png "Dropdown menu showing 3 notifications, and the “See all Notifications” button")
+
+Clicking `See all Notifications` at the bottom will take the user to the complete list of notifications. The same list can be also accessed by opening directly the `/notifications` URL (e.g. [pontoon.mozilla.org/notifications](https://pontoon.mozilla.org/notifications/)).
+
+## Automated notifications
+
+Pontoon includes several types of automated notifications, and most of them can be [manually disabled](users.md#notification-subscriptions) by users.
+
+### New strings
+
+This notification is triggered when new strings are added to a project. The notification is sent to all users who contributed translations to that project, as soon as new strings are available.
```

Comment:
Personal preference: s/triggered/sent. (also below)

Reply:
The "triggered" is to avoid repeating "sent". Rephrasing to avoid both.